content/set-up/clients/client-maintenance.md

```diff
@@ -18,12 +18,30 @@ To edit an existing client:
 
```

Comment:
Same deal with the step above--we need to refer to the dropdown menu with a locater.

Reply:
Feel free to ignore this one based on what I said earlier.